helix-view/src/editor.rs

```diff
@@ -1169,6 +1173,7 @@ impl Editor {
     pub fn refresh_config(&mut self) {
         let config = self.config();
         self.auto_pairs = (&config.auto_pairs).into();
+        self.registers.clipboard_provider = config.clipboard_provider.get_provider();
```

Comment:
I believe this will reset the internal buffers (within NoneProvider for example) when you reload config even if you don't change your clipboard config. I'm not sure there's an easy way to fix this though. We could consider a much larger refactor for clipboard providers to remove state from the clipboard providers (since we store yanks and pastes on the Registers type anyways), and probably remove the clipboard provider trait and just use the enum directly. I'm not sure how large of a change this would take though - I will try out some changes locally and revisit this

Reply:
Yeah, I can think of some gross solutions to this, but this seemed like the least bad option?